ui/controls.js

```diff
@@ -79,6 +79,9 @@ shaka.ui.Controls = function(player, videoContainer, video, config) {
   /** @private {!HTMLMediaElement} */
   this.video_ = this.castProxy_.getVideo();
 
+  /** @private {!HTMLMediaElement} */
+  this.localVideo_ = video;
```

Comment:
How does PiP interact with our Chromecast support, anyway? I don't see any code that disables the PiP button when we are casting, and the .less doesn't hide it either. Well, that's a little outside of the scope of this CL, I guess.

Reply:
@theodab You're probably right about hiding the pip button when casting, I don't think it makes sense in that context. I'll work on that once this PR lands.

@beaufortfrancois Thanks for the PR! Let me play around a bit with it locally to better understand what's going on with events on this.video_ and get back to you.